impl/client/src/main/javascript/web/util/formatting.js

```diff
@@ -192,6 +192,30 @@ define("common-ui/util/formatting", ['common-ui/util/timeutil', 'common-ui/util/
           }
 
           if(timezone == 'client') {
+
+              if (value.length == 28) {
+
+                  var currentDate = new Date();
+                  var offset = -currentDate.getTimezoneOffset();
```

Comment:
Perhaps calling this `offsetClientMinutes` would be clearer.

Reply:
Agree, please see the new updated commit.